v3/integrations/nrecho-v3/nrecho.go

```diff
@@ -93,3 +91,23 @@ func Middleware(app *newrelic.Application) func(echo.HandlerFunc) echo.HandlerFu
 		}
 	}
 }
+
+// WrapRouter extract api endpoints from the echo instance passed to it
+// which is used to detect application URL mapping(api-endpoints) for provable security.
+// Skip if you are not using [nrsecurityagent](https://pkg.go.dev/github.com/newrelic/go-agent/v3/integrations/nrsecurityagent).
```

Comment:
I'd recommend rewording this. Since the `WrapRouter` function is so generically named, it doesn't make sense that it would be only for the security agent (and that would conflict with the use of similarly-named wrappers in other integrations). Instead, we should consider `WrapRouter` a general-purpose wrapper which at the moment happens to only be needed with the security agent.

Proposed wording:

> In this version of the integration, this wrapper is only necessary if you are using the New Relic security agent integration [github.com/newrelic/v3/integrations/nrsecurityagent], but it may be enhanced to provide additional functionality in future releases.

(also note the correct markdown syntax used in go documentation as documented here: https://go.dev/doc/comment#links since the comments are used to generate our official API documentation.)